flake-info/src/bin/flake-info.rs

```diff
@@ -265,7 +265,14 @@ async fn run_command(
                 .collect::<Vec<_>>();
 
             if !errors.is_empty() {
-                return Err(FlakeInfoError::Group(errors));
+
+                if exports.is_empty() {
+                    return Err(FlakeInfoError::Group(errors));
+                }
+                warn!("=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=");
+                warn!("Some group memebers could not be evaluated: {}", FlakeInfoError::Group(errors));
```

Comment:
```suggestion
                warn!("Some group members could not be evaluated: {}", FlakeInfoError::Group(errors));
```

Been looking at too many memes lately?